src/components/tx/TxData.js

```diff
@@ -597,7 +603,7 @@ class TxData extends React.Component {
           </h4>
           <div>
             <span>It is voided by: </span>
-            {renderListWithLinks(this.props.meta.voided_by, true)}
+            {renderListWithLinks(conflictNotTwin, true)}
```

Comment:
question: Why have you changed this?

The transactions exhibited are not correct.
![image](https://github.com/user-attachments/assets/7c59d234-f72b-47a7-9958-e78a4a32074a)

![image](https://github.com/user-attachments/assets/28bb6d4d-80ec-432a-a66a-f2f16c844e8d)

Reply:
ended up being modified along with the renderListWithLinks of the valid return, but it has already been corrected. Waiting for review.🙂